watertap/unit_models/electrodialysis_1D.py

```diff
@@ -64,6 +64,25 @@ class ElectricalOperationMode(Enum):
     Constant_Voltage = 1
 
 
+class PressureDropMethod(Enum):
+    none = 0
+    experimental = 1
+    Darcy_Weisbach = 2
+
+
+class FrictionFactorMethod(Enum):
+    defined = 0
+    Ponzio = 1
+    Gurreri = 2
+    Kuroda = 3
+
+
+class HydraulicDiameterMethod(Enum):
+    defined = 0
+    spacer_specific_area_known = 1
+    conventional = 2
```

Comment:
Adding the corresponding model changes into the documentation would be good.

Reply:
Yes. To do in this quarter.